sql-schema/252.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM * `graph_types` WHERE `graph_descr` = 'HTTP Server Connections Acitve';
```

Comment:
Should be named `251.sql` :)

Reply:
Sorry @laf got confused with another branch I'm working on.